### PR TITLE
Create Mint page

### PR DESCRIPTION
**Pull request by markulrich**

### CODE REVIEW

**Comment by markulrich** [comment]
screenshot_20180620-145433

File: src/shared/IosAndroidSafeAreaView.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+import {
+  Text,
+  View,
+  SafeAreaView,
+  ViewProps,
+  Platform,
+  StatusBar
+} from "react-native";
+
+export const IosAndroidSafeAreaView: React.StatelessComponent<
```
Comment (osdiab) [inline src/shared/IosAndroidSafeAreaView.tsx]: Can we just call it `SafeAreaView`? or `SaferAreaView` lol. Don't like the platform-specific naming.

File: src/components/pages/Mint.tsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+    <IosAndroidSafeAreaView style={{ flex: 1 }}>
+      <View
+        style={[
+          styles.centerFlex,
+          { flexDirection: "row", marginHorizontal: 50 }
+        ]}
+      >
+        <View style={styles.centerFlex}>
+          <Text style={styles.number}>
+            {loggedInMember.balance.minus(30).toString()}
```
Comment (osdiab) [inline src/components/pages/Mint.tsx]: Can you leave a comment saying that this is a placeholder lol.

**Comment by osdiab** [comment]
I think it's not really intuitive as it stands but we can think about copy and the design language later

**Pull request merged**